src/package/Microsoft.TestPlatform.Portable/Microsoft.TestPlatform.Portable.csproj

```
@@ -15,7 +15,8 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <IsPackable Condition="'$(DotNetBuildSourceOnly)' != 'true'">true</IsPackable>
+    <!-- Don't produce this package when building from source or inside the VMR as it relies on a VS license. -->
+    <IsPackable Condition="'$(DotNetBuildSourceOnly)' != 'true' and '$(DotNetBuildOrchestrator)' != 'true'">true</IsPackable>
```

```suggestion
    <IsPackable Condition="'$(DotNetBuild)' != 'true'">true</IsPackable>
```

I think this is more accurate.